API Management error guidance (#227)
bcgov/public-cloud-techdocs: docs/azure/azure-services/api-management.md

!!! success "APIM v2"
    Version 2 of the API Management Service is now available in the Canada Central Azure region. Use the [v2 tiers](https://learn.microsoft.com/en-us/azure/api-management/v2-service-tiers-overview) of APIM to enable network isolation and private endpoints.

## Required protocols

The Landing Zones enforce the `API Management APIs should use only encrypted protocols` policy, requiring all APIs to use encrypted protocols. Supported protocols are **HTTPS** and **WSS**.

If you attempt to create an API in API Management using an unsupported protocol, the UI shows _no error message_ but the API creation fails.

> To confirm this is the reason why your API creation is failing, check the Activity Logs, which will show that the `Create API or Update API` action was denied, and the message will indicate that the API does not use a supported protocol.

Reviewer: One thought here: Users will see the same non-compliance message for every policy in `Enforce Public Ingress security best practices`, so to see that the deployment failed on `API Management APIs should use only encrypted protocols` specifically, they'll still need to crack open the JSON.

Suggestion to tweak the wording slightly:
```suggestion
To confirm this is the reason why your API creation is failing, check the Activity Logs, which will show that the `Create API or Update API` action was denied. The JSON details for the Deny action will indicate that the API does not use a supported protocol.
```

<img width="1516" height="743" alt="Screenshot 2026-07-28 at 3 55 38 PM" src="https://github.com/user-attachments/assets/48f3a52a-cee0-4fba-bd98-6c4fc72ee495" />
